app/grandchallenge/reader_studies/forms.py

```diff
@@ -299,6 +312,20 @@ def __init__(self, *args, **kwargs):
             )
         )
 
+    def save(self, *args, **kwargs):
+        instance = super().save(*args, **kwargs)
+        if "widget" in self.changed_data:
+            new_widget = self.cleaned_data["widget"]
+            try:
+                old_widget = instance.widget
```

Comment:
I realized now that this does not work. `instance.widget` == `new_widget`. I'll need to find a different way of getting the preious widget value

Reply:
Not setting the `question` on the `widget_instance` in the clean method should do the trick.